straxen/plugins/nveto_recorder.py

```diff
@@ -114,8 +114,9 @@ def compute(self, raw_records_nv, start, end):
 
         lr = strax.sort_by_time(lr)
         strax.zero_out_of_bounds(lr)
+        baseline_samples = straxen.get_correction_from_cmt(self.run_id, self.config['nbaseline_samples_lone_records_nv'])
```

Comment:
Could you please move this line into a
```
def setup(self):
    ....
```
Like n corrected areas, if you do not have time no problem then I change it.

Reply:
done